bosun.md

```diff
@@ -0,0 +1,44 @@
+# Steering Committee Bosun Guide  
+
+This doc is a guide to help the delegated Steering Committee (SC) Member bosun the public meeting. SC members will pick a new one for each meeting.  
+
+## Before
+1. Create the skeleton meeting agenda from follow up items the meeting prior, parking lot topics at the top, and ask other steering members to populate.
+2. Read through the agenda just to familiarize yourself with the content to assign timeboxes. [repeat a few times in the days leading to the meeting if it’s a live agenda]
+3. Post the draft agenda at least 3 days before with a call for final topics and a clear timebox to the kubernetes-dev mailing list. This will act as a reminder and put the agenda on the community’s radar, too.
+- Note that the dev mailing list is the source of truth. While a Slack notification is nice and helpful, the mailing list is the main communication channel.
+- Also post to the #steering-committee, #kubernetes-dev, #chairs-and-techleads slack channels
+6. Make sure there’s someone who volunteers from contribex to act as the meeting host to mute folks, verify recording, note taker and other zoom admin tasks.
+7. If no one steps up for note taking, you’re it.
+8. Do we want to poll the attendees with any questions? Do we want to take any live community voting for pulse checks? (not decisions) zoom has these features!
+9. DAY OF: Join ten minutes early to make sure all audio, video, and zoom administration (including capability to record) is right.
+10. Make sure that you have cleared your desktop and can be prepared to present if needed. Sometimes AV problems will require you to step in and present.
+
+## During
+### Starting:
+1. Before recording the meeting, tell the group “This is a Kubernetes meeting that will be posted publicly on YouTube, so please be mindful that what you say is being recorded.
+2. Please also mute if you are not speaking.” Talk about the process of waiting for n Steering members to arrive before you start.
```

Comment:
nit: maybe a quick link to https://github.com/kubernetes/steering/blob/master/charter.md#quorum for easy discoverability on the numbers?